example/src/screens/calendarsList.js

```diff
@@ -25,16 +25,10 @@ const CalendarsList = () => {
 
         return (
           <View
-            style={{
-              flexDirection: 'row',
-              width: '100%',
-              justifyContent: 'space-between',
-              marginTop: 10,
-              marginBottom: 10
-            }}
+            style={styles.header}
           >
-            <Text style={{marginLeft: 5, ...textStyle}}>{`${month}`}</Text>
-            <Text style={{marginRight: 5, ...textStyle}}>{year}</Text>
+            <Text style={[styles.month, {...textStyle}]}>{`${month}`}</Text>
```

Comment:
Spreading a object inside an empty object (`{...textStyle}`) just creates a new object each time and breaking reference comparison.

Reply:
True. Done